lib/valueflow.cpp

```diff
@@ -1221,17 +1221,37 @@ static void valueFlowSameExpressions(TokenList *tokenlist)
     }
 }
 
+static bool isIncompleteVar(const Token *tok)
+{
+    if (!Token::Match(tok, "%name%"))
+        return false;
+    if (tok->tokType() != Token::eName)
+        return false;
+    if (tok->flags() != (1 << 20))
+        return false;
+    if (Token::Match(
+            tok,
+            "audit|axiom|alignas|alignof|asm|auto|catch|char|class|concept|const|consteval|constexpr|co_await|co_return|co_yield|decltype|default|do|enum|explicit|export|extern|friend|inline|mutable|namespace|new|noexcept|private|protected|public|reflexpr|register|requires|sizeof|static|static_assert|struct|synchronized|template|this|thread_local|throw|try|typedef|typeid|typename|union|using|virtual|void|volatile|override|final"))
```

Comment:
if we handle c++20 keywords I guess somewhere it should be checked that C++20 is actually used.

Reply:
I would suggest to put these keywords in some set(s).